db/seeds.rb

```diff
@@ -7,7 +7,7 @@
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
 require 'date'
-user = User.create(email: 'user@email.com', name: 'User Name', password: 'password', birthday: Date.yesterday)
+user = User.create(email: 'user@email.com', name: 'User Name', password: 'password', birthday: Date.yesterday.ago(172800))
```

Comment:
I'd replace `Date.yesterday.ago` with `2.days.ago` or `20.years.ago`. Easier to understand